tiaas/settings.py

```diff
@@ -116,6 +116,7 @@
 
 # Default period to retain contact information (months)
 TIAAS_SHOW_ADVERTISING = True
+TIAAS_GDPR_AUTO_REDACT = False
```

Comment:
```suggestion
TIAAS_GDPR_AUTO_REDACT = True
```

if it's ok, I'd like to keep it defaulting to the old behaviour (redaction) with the possibility for sites to opt in to the new behaviour.